tests/detabler/detabler.js

```diff
@@ -0,0 +1,198 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: contextmenu,detabler */
+
+(function () {
+	'use strict';
+    bender.editor = {
+        config: {
+            enterMode: CKEDITOR.ENTER_P
+        }
+    };
+    bender.test({
+        assertHtml: function (expected, actual, msg) {
+            assert.areEqual(bender.tools.fixHtml(expected), bender.tools.fixHtml(actual), msg);
+        },
+		setUp: function () {
+			this.command = this.editorBot.editor.getCommand('detable');
+		},
+        'it does not do anything outside of tables': function () {
+            var editor = this.editorBot.editor,
+                startHtml,
+                endHtml;
+
+            startHtml = '<p>foobar^</p>' +
+                '<p>baz</p>' +
+                '<table>' +
```

Comment:
was tempted to put the whitespace inside the string literals so that auto-indent fixer doesn't break it.  maybe do this later.